packages/interwovenkit-react/src/components/Collapsible.module.css

```
@@ -0,0 +1,24 @@
+.button {
+  justify-content: flex-start;
+  align-items: center;
+  gap: 2px;
+
+  width: fit-content;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)